src/muxer/yamux/yamuxed_connection.cpp

```diff
@@ -9,6 +9,9 @@
 #include <libp2p/muxer/yamux/yamux_frame.hpp>
 #include <libp2p/muxer/yamux/yamux_stream.hpp>
 
+#define TRACE_ENABLED 0
```

Comment:
It is good to have such a way to debug. Despite this, I suggest to remove that define from the production-ready source code.

Reply:
to be discussed